ipn/ipn_clone.go

```diff
@@ -24,10 +24,7 @@ func (src *Prefs) Clone() *Prefs {
 	*dst = *src
 	dst.AdvertiseTags = append(src.AdvertiseTags[:0:0], src.AdvertiseTags...)
 	dst.AdvertiseRoutes = append(src.AdvertiseRoutes[:0:0], src.AdvertiseRoutes...)
-	if dst.Persist != nil {
```

Comment:
Hey @maisem this is what happens if I run `go generate` in `ipn/`, I presume this is not intentional. Want me to put it back and ignore the go-generate github check?

Reply:
This looks correct. Previously `Persist` was a value type and didn't contain any pointers or slices so it was safe to pass around by just doing a shallow copy. With your change it now has mutable data so it requires more protection around it which means that it needs to return a PersistView from the call to `PrefsView.Persist()`.